### ISSUE TITLE
sql formatting never returns

### ISSUE THREAD

**Issue by qdx** [andialbrecht/sqlparse#184]
The formatter never returns for the following sql query.

```
SELECT *
FROM tables
WHERE tables.id > 1 -- Test Test
      LIMIT 100;
```

**Comment by andialbrecht** [comment]
hm, maybe that was just an temporary issue on sqlformat.org? Running this statement through the formatter shows no issues.

Could you please try again?

**Comment by qdx** [comment]
Hey @andialbrecht , sorry but I've simplized the original query to try to narrow down the problem, the original query looks like this:
```
SELECT F.id,
       PL.id
FROM ftables AS F
JOIN ctables AS CMP ON CMP.id = F.c_id
JOIN stables AS SP ON F.s_id = SP.id
JOIN ptables AS PL ON PL.f_id = F.id
WHERE F.deleted = 0
  AND F.start_date < NOW()
  AND F.end_date > NOW()
  AND CMP.a_id != 44 -- Test
  AND F.f_t_id NOT IN (5, 6, 7)
  AND F.i_b = 1
  AND SP.id NOT IN (2, 8, 9, 10, 12, 13, 15, 16, 17)
  AND PL.deleted = 0
  AND PL.cancelled = 0 LIMIT 100;
```
Although the simplized query now seems to be fine, the original one is stilling timeout when I tried.
P.S. I've changed some of the table/column names, since this is a query from our production environment.

**Comment by andialbrecht** [comment]
woot, great thanks! That was the SQL I was looking for!!

I've noticed a strange behavior for some statements but couldn't find a statement to reproduce it until now. 

For the record, the wrong behavior was introduced in this commit: 8c5c0684ac61ab7d0c5e77881728c8106f2877f9

**Comment by andialbrecht** [comment]
Fixed in ab827eaf935aba776715c4693ec9f320008a6708

Thanks again for reporting!
